socrates_api/tasks.py

```diff
@@ -795,6 +795,8 @@ def ipmi_power_state(self, asset):
             return ret
         elif asset['asset_subtype'] == 'libvirt':
             return ipmi_power_state_libvirt(asset)
+    else:
```

Comment:
I think this should be inside of the asset_type server block, as otherwise things like storage will report missing oob.

Reply:
Moved it around a bit, shouldn't report missing oob for other asset_types this way.